bitsail-connectors/connector-redis/src/test/java/com/bytedance/bitsail/connector/redis/sink/RedisSinkTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.redis.sink;
+
+import com.bytedance.bitsail.common.BitSailException;
+import com.bytedance.bitsail.common.exception.CommonErrorCode;
+import com.bytedance.bitsail.connector.redis.core.jedis.JedisCommand;
+import com.bytedance.bitsail.connector.redis.core.jedis.JedisCommandDescription;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class RedisSinkTest {
```

Comment:
can we add a test for type conversion

Reply:
because we almost do nothing in type convert, what i do is only read the different field as string, for example, in the hash type, we need to insert key, hash_key, hash_value, we only need to consider all this field as string, and use jedis to write it.